dist/utils/index.mjs

```diff
@@ -0,0 +1,6 @@
+import 'qs';
+export { D as base64Encode, e as camelCase, c as cloneDeep, d as defaultMethods, w as getLocationParams, j as getOptions, n as isEmpty, g as isEqual, k as isFunction, o as isLiveMode, b as isObject, i as isServer, l as loadScript, q as map, m as merge, p as pick, r as reduce, x as removeUrlParams, s as set, E as setOptions, f as snakeCase, C as stringifyQuery, a as toCamel, h as toCamelPath, F as toLower, t as toSnake, B as trimBoth, y as trimEnd, A as trimStart, u as uniq, v as vaultRequest } from '../index.738bc14a.mjs';
```

Comment:
The /dist files should not be part of this PR

Reply:
Done